src/provider/application.js

```diff
@@ -76,6 +79,17 @@ class Application extends EventEmitter {
     );
   }
 
+  /**
+   * imageRequestResize function to call requestResize event for slow loading image
+   * @param {object} event - event which triggered the listener
+   */
+  imageRequestResize(event) {
+    const tgt = event.target;
+    if (this.resizeConfig && tgt.tagName === 'IMG' && !(tgt.hasAttribute('height') || tgt.hasAttribute('width'))) {
```

Comment:
resizeConfigi check should go in the requestResize function. It shouldn't execute until its "been initialized" by receiving config